code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -34,8 +34,11 @@
 			src << "<span class='warning'>Your powers are useless on this holy ground.</span>"
 			return 0
 	if(check_holy(src) && !fullpower)
-		var/turf/simulated/T = get_turf(src)
-		if(T.lighting_lumcount > 2)
+		var/atom/movable/lighting_overlay/L
+		for(var/atom/movable/lighting_overlay/LF in get_turf(src))
```

Comment:
I believe finding things like all of a typepath in world may be faster with locate(), for example shuttle movement locating an entire area. However if you can use any method to get a direct reference to the object though variables or list of references containing the object it would be much cheaper to do so. I couldn't really say which would be faster in this specific instance.